run

```diff
@@ -10,6 +10,10 @@ else
   rm /tmp/server-*.pem
 fi
 
+# This is a hack due to musl libc's DNS handling
+cat /etc/resolv.conf | head -n 3 > dns
+cat dns > /etc/resolv.conf
```

Comment:
This would always take the first three lines, nobody guarantees that it includes any dns server..
But good point, I just ran into a related issue and also realize that this is a problem on DCOS. We probably have to change this to something glibc based.. Either way, let's address this in a separate PR.